GenderPayGap.WebUI/Content/Site.css

```diff
@@ -663,6 +663,10 @@ input[type=submit] {
     height: 20px;
 }
 
+.finder .govuk-option-select .options-container .option-select-legend {
+    display: none;

```

Comment:
`display: none;` hides the content from the screen readers as well so that might be the reason why you've seen no difference. You can read [here](https://webaim.org/techniques/css/invisiblecontent/) about different techniques for hiding the content. 
It also seems that some screen readers ignore the `legend` tags. However. even if it's optional it's best practice to have a legend for each field set to describe the field set data.

Reply:
This is really helpful! Thanks Malina :) .